mlir/utils/generateE2ETest.py

```diff
@@ -117,6 +117,9 @@ def usage():
                 if "exclude" in test:
                     exclusions = generate_option_list(axis_prefixes, test, "exclude", "values")
                 for opt in combinations:
+                    # Only generate i8 data type for fwd convolutions
+                    if "i8" in opt[2] and ("conv2d_" in opt[0]):
```

Comment:
Let's more precisely exclude `conv2d_bwd_weight` and `conv2d_bwd_data` or whatever the strings are